lifecycle_test.go

```diff
@@ -195,22 +206,25 @@ func uploadBuilds(t *testing.T) func() {
 func createRbBackwardCompatible(t *testing.T, specName, sourceOption, rbName, rbVersion string, sync bool) {
 	specFile, err := getSpecFile(specName)
 	assert.NoError(t, err)
-	createRb(t, specFile, sourceOption, rbName, rbVersion, sync)
+	createRb(t, specFile, sourceOption, rbName, rbVersion, sync, false)
 }
 
-func createRbFromSpec(t *testing.T, specName, rbName, rbVersion string, sync bool) {
+func createRbFromSpec(t *testing.T, specName, rbName, rbVersion string, sync bool, withoutKey bool) {
 	specFile, err := tests.CreateSpec(specName)
 	assert.NoError(t, err)
-	createRb(t, specFile, "spec", rbName, rbVersion, sync)
+	createRb(t, specFile, "spec", rbName, rbVersion, sync, withoutKey)
 }
 
-func createRb(t *testing.T, specFilePath, sourceOption, rbName, rbVersion string, sync bool) {
+func createRb(t *testing.T, specFilePath, sourceOption, rbName, rbVersion string, sync bool, withoutKey bool) {
```

Comment:
withoutKey --> withoutSigningKey

(to make it easier for the random reader to understand what this argument is).